js/mail.js

```diff
@@ -66,6 +151,28 @@ var Mail = {
 		}
 	},
 	UI: {
+		changeFavicon: function (src) {
+			$("link[rel='shortcut icon'").attr("href",src);
```

Comment:
The notifications and check doesn’t work in Firefox seemingly due to this selector:
`Error: Syntax error, unrecognized expression: link[rel='shortcut icon' jquery.min.js:4:15709`

@DeepDiver1975